content/docs/command-reference/add.md

```diff
@@ -56,13 +63,15 @@ more details.
 > treated as _changed_ by `dvc repro`, which always executes them. See `dvc run`
 > to learn more about stage files.
 
-By default DVC tries to use reflinks (see
+By default, DVC tries to use reflinks (see
 [File link types](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
 to avoid copying any file contents and to optimize DVC-file operations for large
 files. DVC also supports other link types for use on file systems without
 `reflink` support, but they have to be specified manually. Refer to the
 `cache.type` config option in `dvc config cache` for more information.
 
+### Tracking directories
```

Comment:
let's remove `--recursive` below, especially it should not be going first (may be only mention that in certain cases -`--recursive` might be used, etc)

Reply:
OK well I just inverted the order of these numbered items for now.